pnc-ui/app/keycloak.json

```diff
@@ -0,0 +1,21 @@
+{
+  "realm" : "PNC.REDHAT.COM",
+  "resource" : "pnc-web-local",
+  "realm-public-key" : "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCF6WP9SC6i9ulVqAItdu1qhsN9LoIywlJ3Qw7RUIMmPjKToCnDoq8glkJAkS4I3PbpWaFSBRvSObmAm4XvGv69kl1UPUW76Wgyqvr09fNSNds34MV4NuULE1muHHj3x49RqH4J/UPQvICtE77t+lvjd526uqHPyn+mZlN4B0K6hQIDAQAB",
+  "auth-server-url" : "https://keycloak-pncauth.rhcloud.com/auth",
+  "ssl-required" : "none",
+  "use-resource-role-mappings" : false,
+  "enable-cors" : true,
+  "cors-max-age" : 1000,
```

Comment:
We should keep REST and WEB in different WARs so that you can use (also build) REST without a WEB UI.